recipe/meta.yaml

```diff
@@ -43,13 +41,13 @@ about:
   home: https://github.com/python-versioneer/python-versioneer
   license: Unlicense
   license_file: LICENSE
-  license_family: None
+  license_family: OTHER
```

Comment:
![SmartSelect_20231220_073801_Firefox](https://github.com/AnacondaRecipes/versioneer-feedstock/assets/1415602/347a8a51-fc24-4a25-896a-eb2a93b50f06)

Versioneer uses the unlicense, is that really "other" for these purposes?

Reply:
fixed